lib/concentrate/parser/gtfs_realtime_enhanced.ex

```diff
@@ -119,13 +119,32 @@ defmodule Concentrate.Parser.GTFSRealtimeEnhanced do
         load: Map.get(vp, "load"),
         capacity: Map.get(vp, "capacity"),
         occupancy_status: Map.get(vp, "occupancy_status"),
-        occupancy_percentage: Map.get(vp, "occupancy_percentage")
+        occupancy_percentage: decode_occupancy_status(vp)
```

Comment:
Are `_status` and `_percentage` mixed up here?

Reply:
whoops!